docker-compose.yml

```diff
@@ -41,6 +41,7 @@ services:
       AUTHENTIK_POSTGRESQL__USER: ${PG_USER:-authentik}
       AUTHENTIK_POSTGRESQL__NAME: ${PG_DB:-authentik}
       AUTHENTIK_POSTGRESQL__PASSWORD: ${PG_PASS}
+      AUTHENTIK_POSTGRESQL__USE_PGBOUNCER: false
```

Comment:
As this is the same value as the default, imo it shouldn't be set here to keep the compose file small (it's already a bit much)